rfc/20251211-contribution-docs.md

```diff
@@ -0,0 +1,75 @@
+
+[^2]: "[Casual contributors] do not want to spend time familiarizing themselves with a project's ... contribution process." - "Working in Public", Nadia Eghbal
+[^3]: [More Common Than You Think: An In-depth Study of Casual Contributors, 2016](https://ieeexplore.ieee.org/document/7476635)
+[^4]: [Communication and Code Dependency Effects on Software Code Quality, 2022](https://arxiv.org/pdf/1904.09954)
+
+In light of these facts, this RFC recommends these standards for our documents and issues:
+
+ * Contributing documentation should be standardized, both within OpenTofu and relative to the wider open source community
+   * We want to make it as smooth as possible to onboard onto OpenTofu's core and its satellite projects, aimed towards the casual contributor
+ * Issue labels should be standardized, both within OpenTofu and relative to the wider open source community
+ * Provide the full breadth of "contributable" repositories, and prepare those repositories for contributions
+   * If someone wanted to contribute to this project, they may only know of `opentofu/opentofu`, if even that. It may not be clear that one could contribute to the language server, or website, or even brand artifacts.
+
+## Short-Term goals
+
+In order to best serve these standards, we recommend the following course of action:
+
+### Determine which repositories can be contributed to
+
+https://github.com/opentofu/org/issues/23
+
+For lack of a better term, a "supervised" repository is one for which a non-maintainer community member can provide contributions and expect a maintainer to review and potentially incorporate. These repositories should be explicitly listed in the contribution documents as "open to volunteers."
+
+### Contributing guidelines should be standardized across supervised repositories
+
+https://github.com/opentofu/org/issues/25
+
+A casual contributor should, at a glance, be able to understand all expectations and limitations for their contributions. This includes, but is not limited to, our stance on AI and the scope of prohibition on external code.
+
+Note that none of those guidelines are specific to any development. We may have some development guidelines which are in a supervised repository's `CONTRIBUTING.md`. However, if those were moved to their own document, then `CONTRIBUTING.md` could be standardized and mostly identical across all supervised repositories.
+
+This also means that [any fixes](https://github.com/opentofu/org/issues/26) done to `CONTRIBUTING.md` and `MAINTAINERS.md` could propagate across all relevant repositories.
```

Comment:
Reading in https://github.com/opentofu/org/issues/25 too, I want to ask if syncing these documents manually is the right approach.
I am not sure what the end goal is for the content of these documents, but for the contributing one, I would expect to have repo-specific information too.
Hence, I would suggest to have one globally applicable, general contribution guide in our [org repo](https://github.com/opentofu/org) and have that referenced in all the supervised repositories as the main document. Anything else that might be mentioned in the supervised repository contribution guide should be made clear that are amendments to the main document.
This way we will not go through the, possible, pain of trying and not override repository specifics.